mobidam-sst-management-frontend/frontend/src/components/AddPersonDialog.vue

```diff
@@ -61,12 +61,13 @@
                     <v-text-field
                         ref="address"
                         v-model="zuordnung.funktionsadresse"
-                        label="Funktionsadresse"
+                        label="Funktionspostfach"
                         hint="Welchem Gruppenpostfach gehört diese Person an?"
```

Comment:
Wie lautet das Funktionspostfach?

Reply:
This means _hint="Wie lautet das Funktionspostfach?"_   ?